packages/devextreme/testing/testcafe/helpers/testAccessibility.ts

```diff
@@ -0,0 +1,94 @@
+import createWidget, { WidgetName } from './createWidget';
+import { a11yCheck, A11yCheckOptions, ElementContext } from './accessibilityUtils';
+import { isMaterialBased } from './themeUtils';
+
+export interface Options {
+  [key: string]: any[];
+}
+
+export interface Configuration {
+  component: WidgetName;
+  options?: Options;
+  a11yCheckConfig?: A11yCheckOptions;
+  selector?: ElementContext;
+  created?: (t?: any, optionConfiguration?: Options) => Promise<void>;
+  after?: (t?: any, optionConfiguration?: Options) => Promise<void>;
+}
+
+const defaultSelector = '#container';
+const defaultOptions = {};
+const defaultCreated = async () => {};
+const defaultAfter = async () => {};
+const defaultA11yCheckConfig = isMaterialBased() ? {
+  runOnly: 'color-contrast',
+} : {};
+
+const generateConfigurations = (
+  options: Options,
+  index = 0,
+  prevConfigurations: Options[] = [],
+) => {
+  const keys = Object.keys(options);
+
+  if (index >= keys.length) {
+    return [Object.assign({}, ...prevConfigurations)];
```

Comment:
Let's use ES6 syntax
```suggestion
    return [{ ...prevConfigurations }];
```

Reply:
These expressions are not equivalent. prevConfigurations contains an array. If you use Spread syntax, you will get an object with keys 0, 1, 2 - because an array, as an object, has such keys.

Reply:
Oh, sure, I see